src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -974,8 +985,8 @@ void CountryInstance::update_modifier_sum(Date today, StaticModifierCache const&
 	modifier_sum.add_modifier(static_modifier_cache.get_infamy(), country_source, infamy);
 	modifier_sum.add_modifier(static_modifier_cache.get_literacy(), country_source, national_literacy);
 	modifier_sum.add_modifier(static_modifier_cache.get_plurality(), country_source, plurality);
-	// TODO - difficulty modifiers, war, peace, debt_default_to, bad_debter, generalised_debt_default,
-	//        total_occupation, total_blockaded, in_bankrupcy
+	// TODO - difficulty modifiers, war, peace, debt_default_to, bad_debtor, generalised_debt_default,
```

Comment:
Good catch on these typos, some of them are copying typos in the static modifier names vic2 uses but the variables in `StaticModifierCache` that they're mapped to still use correctly spelt names, so it's how we should be refering to them 👍

Reply:
I have a spellchecker for code since I have dyslexia, I just cleaned these up to get rid of "unknown word" errors. But I'm glad they were actual typos, I made sure to download the UK english dictionary so I don't edit any UK spellings